pkg/ebpf/c/common/buffer.h

```diff
@@ -29,169 +30,171 @@ statfunc buf_t *get_buf(int idx)
 // biggest elem to be saved with 'save_to_submit_buf' should be defined here:
 #define MAX_ELEMENT_SIZE sizeof(struct sockaddr_un)
 
-statfunc int save_to_submit_buf(event_data_t *event, void *ptr, u32 size, u8 index)
+statfunc int save_to_submit_buf(args_buffer_t *args_buf, void *ptr, u32 size, u8 index)
```

Comment:
I think the name gives it needed context, as we use "buffer" in other contexts as well (the event buffer and scratch buffer for example).

Reply:
I think this change makes sense anyway. All the helpers used to save to the buffer are called save_XXX_to_buf, so it is clear we are talking about the "args buffer". There is also a name incosistency in this function since it is called save_to_**submit_buf** while here we call it args buf, so this renaming will solve this problem as well. And it also has the advantage of keeping the code nit and clean

Reply:
Done.